### PR TITLE
Fix Combiner Load Test

### PR DESCRIPTION
**Pull request by soloseng**
### Description

Fixed a bug where load testing always returned a timeout even when successful.

### Other changes

Switch to using `yargs`, and added a flag to easily run serial load tests.

### CODE REVIEW

**Comment by soloseng** [comment]
Related to #10334  , but does may not resolve the issue.

File: packages/phone-number-privacy/monitor/src/scripts/run-load-test.ts [hunk 1]
```diff
@@ -1,31 +1,54 @@
...
-    blockchainProvider = 'https://forno.celo.org'
-    break
-  default:
-    printHelpAndExit()
-    break
-}
-
-concurrentLoadTest(Number(args[1]), blockchainProvider!, args[0] as OdisContextName) // tslint:disable-line:no-floating-promises
+// tslint:disable-next-line: no-unused-expression
+yargs
```
Comment (alecps) [inline packages/phone-number-privacy/monitor/src/scripts/run-load-test.ts:34]: Nice!!

File: packages/phone-number-privacy/monitor/src/scripts/run-load-test.ts [hunk 2]
```diff
@@ -1,31 +1,54 @@
...
+        .positional('numWorkers', {
+          type: 'number',
+          description: 'Number of machines that will be sending request to ODIS.',
+        })
+        .option('isSerial', {
+          type: 'boolean',
+          description: 'run test workers in series.',
+          default: false,
+        }),
+    (args) => runLoadTest(args.contextName!, args.numWorkers!, args.isSerial)
```
Comment (alecps) [inline packages/phone-number-privacy/monitor/src/scripts/run-load-test.ts:58]: does casting these this way cause weird type errors when the parameters aren't provided? Ideally we'd show the help in this case. Looks like that's probably already what happens but just want to double check

**Review by alecps — APPROVED** [review]
Looks great!! Thanks!

File: packages/phone-number-privacy/monitor/src/scripts/run-load-test.ts (hunk 2 above)
Comment (soloseng) [reply packages/phone-number-privacy/monitor/src/scripts/run-load-test.ts:58]: positional args are required, so not including them would show the usage

I also added a check for numWorkers <1

**Pull request merged**